tests/microos/image_checks.pm

```diff
@@ -42,6 +42,9 @@ sub run {
         $left_sectors = 1792;
         record_soft_failure "bsc#1220722: no unpartitioned space left on aarch64";
     }
+    if (is_leap_micro("5.4+") && is_aarch64) {
```

Comment:
I believe this is the same as L#36.

Anyway, this whole left sectors logic should be refactored. Because it has become a mess. If you wish to, it would be nice to make it an if-elsif-else block.

Reply:
L#36 checks with is_sle_micro, whereas I used is_leap_micro. I'll try improving readability.

Reply:
As I have mentioned, the gap between sle and leap family has been addressed and should have the same properties as much as possible. In this particular case, if sle-micro('x') has left over sectors y, the same should be expected in leap-micro('x').

Reply:
Fixed in https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/f677e5e3ff996a5ea69fa004d48d074c08e1083d